CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## XX.XX.XX - 20XX-XX-XX
 
+### PaymentSheet
+* [ADDED][8595](https://github.com/stripe/stripe-android/pull/8595) PaymentSheet now supports Sunbit for PaymentIntents in private beta.
```

Comment:
can/should we link to info about the beta in case merchants are interested in joining? Not sure if that makes sense for private beta or not

Also a nit, but can you match the wording we used for Multiblanco in the changelog for consistency?

Reply:
Good question. Generally, we're the ones who reach out to users for Private betas, but if someone was excited to use it and wrote into Stripe, we could add them (assuming they met our criteria).

I removed the private beta wording to follow Multibanco since our release cycle isn't super relevant to this change.